src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
> I agree it's already bad but don't make it worse by introducing another variant.

This variant exists in the code, and it's IMO better than the rest.

> No, whats the need?

All above. It's about clarity and predictability. Things are where they belong.